CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ set(SOURCE_FILES
         src/shaka_scheme/system/base/Bytevector.cpp
         )
 
-add_library(${SHAKA_SCHEME_LIBRARY_NAME} SHARED ${SOURCE_FILES})
```

Comment:
This line does not belong in this CMakeLists.txt, look for the file that defines the `SOURCE_FILES`, the `src/shaka_scheme/system/gc/GCNode.hpp` line should go in that file. Please make this change and push it to your GcNode branch. Any new changes that you push will automatically reflect on this pull request.